pkg/operator/ceph/cluster/mon/drain.go

```diff
@@ -97,3 +119,18 @@ func (c *Cluster) allowMonDrain(request types.NamespacedName) error {
 	}
 	return nil
 }
+
+func UsePDBV1Version(Clientset kubernetes.Interface) (bool, error) {
```

Comment:
`mon` package does not look like a good place for this method. Using `mon.UsePDBV1Version` sounds a bit off.  Maybe `k8sUtil` is a better place for this method.